src/pages/deriv-go/_other-apps.tsx

```diff
@@ -125,7 +123,7 @@ const OtherApps = () => {
                         {other_apps.map((item, index) => {
                             if (is_uk_eu && index == 1) return
                             return (
-                                <Card key={item.title}>
```

Comment:
Is there any reason changing this key? because using index as key is an anti-pattern due to this source: https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9